apps/condo/lang/ru.json

```diff
@@ -234,8 +234,6 @@
   "pages.condo.property.type.village": "Коттеджный поселок",
   "pages.condo.property.unit.Name": "Название",
   "pages.condo.property.warning.modal.HelpMessage": "Информация о доме не сохранится, если вы уйдёте со страницы. Сохраните изменения или продолжите редактировать информацию о доме, закрыв это сообщение.",
-  "pages.condo.property.warning.modal.AddressValidationErrorMsg": "Вы не выбрали сооружение или тип выбранного сооружения не поддерживается",
```

Comment:
translations is not synced

Reply:
It seems, like @MrFoxPro has resolved it in `025e9759`.